src/Nox.ClientApi.Tests/Tests/Controllers/WorkplacesControllerTests.cs

```diff
@@ -309,6 +309,44 @@ public async Task Delete_AllRefToCountry_Success()
 
         #endregion DELETE Delete all ref to related entity /api/{EntityPluralName}/{EntityKey}/{RelationshipName}/$ref => api/workplaces/1/Country/$ref
 
+        #region DELETE Related Entity /api/{EntityPluralName}/{EntityKey}/{RelationshipName} => api/workplaces/1/Country
+
+        [Fact(Skip = "DeleteBehavior.ClientSetNull needs to be verified")]
+        public async Task Delete_Country_Success()
+        {
+            // Arrange
+            var workplaceResponse = await PostAsync<WorkplaceCreateDto, WorkplaceDto>(Endpoints.WorkplacesUrl,
+                new WorkplaceCreateDto { Name = _fixture.Create<string>() });
+
+            // Act
+            var headers = CreateEtagHeader(workplaceResponse?.Etag);
+            var postToCountryResponse = await PostAsync<CountryCreateDto, CountryDto>(
+                $"{Endpoints.WorkplacesUrl}/{workplaceResponse!.Id}/{nameof(WorkplaceDto.Country)}",
+                new CountryCreateDto() { Name = _fixture.Create<string>() },
+                headers);
+
+            headers = CreateEtagHeader(postToCountryResponse?.Etag);
+            var deleteCountryResponse = await DeleteAsync($"{Endpoints.WorkplacesUrl}/{workplaceResponse!.Id}/{nameof(WorkplaceDto.Country)}", headers);
+
+            const string oDataRequest = $"$expand={nameof(WorkplaceDto.Country)}";
+            var getWorkplaceResponse = await GetODataSimpleResponseAsync<WorkplaceDto>($"{Endpoints.WorkplacesUrl}/{workplaceResponse!.Id}?{oDataRequest}");
+            var getCountryResponse = await GetAsync($"{Endpoints.CountriesUrl}/{postToCountryResponse!.Id}");
+
+            //Assert
+            deleteCountryResponse.Should().NotBeNull();
+            deleteCountryResponse!.StatusCode.Should().Be(HttpStatusCode.NoContent);
+
+            getWorkplaceResponse.Should().NotBeNull();
+            getWorkplaceResponse!.Id.Should().BeGreaterThan(0);
+            getWorkplaceResponse!.CountryId.Should().BeNull();
```

Comment:
@rochar CountryId is not set to null as expected by DeleteBehavior.ClientSetNull. Need to investigate in a separate ticket.